.github/workflows/release.yml

```diff
@@ -0,0 +1,41 @@
+name: release
+
+on:
+  push:
+    branches:
+      - release
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: latest
+          cache: "npm"
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Run tests
+        run: npm run test
+
+      - name: Bump version
+        run: |
+          git config --local user.email "action@github.com"
+          git config --local user.name "GitHub Action"
+          npm version patch -m "Bump version to %s"
```

Comment:
very interesting! Where does the value for %s come from? Does it pull from the patch command?

Reply:
Honestly I just asked Claude for help with this since I was interested. It says it comes from the output of npm version, and it knows to do a string replace there with the output. 🤷‍♂️

Reply:
Good ol' Claude! Yeah, ran npm version patch on my machine and it outputs to stdout the version it bumped to. The -m just is a custom message for the commit it creates when creating the tag

Reply:
I think I'd prefer it without the custom message tbh. 

Just npm version patch. The commit will just be `v0.1.11`

Reply:
@rphovley @Tanner-Scadden why not add packages caching to this also, it can speed up the workflow?
https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/caching-dependencies-to-speed-up-workflows#example-using-the-cache-action

Reply:
Package caching would be excellent for optimizing the build process